src/main/java/Bob.java

```diff
@@ -0,0 +1,299 @@
+     *
+     * @param list   a Task class list.
+     * @param object a Task class object.
+     */
+    public static void addTaskToList(Task[] list, Task object) {
+        int index = Task.getCount() - 1;
+        list[index] = object;
+
+        System.out.println("\t" + object);
+        printlnTab(MESSAGE_TASK_COUNT + Task.getCount());
+    }
+
+    /**
+     * Creates a new undated task and appends it to the task list.
+     *
+     * @param list    a Task class list.
+     * @param command the details of the new undated task to be created.
+     */
+    public static void addToDosTask(Task[] list, String command) {
+        printBorder();
+        String[] commandToken = parseCommand(command, DELIMIT_COMMAND);
+
+        if (commandToken[1] != null) {
+            Task temp = new ToDos(commandToken[1]);
+
+            addTaskToList(list, temp);
+            printBorder();
+        }
+        else {
+            printError(MESSAGE_INVALID_ARGC);
+        }
+    }
+
+    /**
+     * Creates a new dated deadline task and appends it to the task list.
+     *
+     * @param list    a Task class list.
+     * @param command the details of the new dated task to be created.
+     */
+    public static void addDeadlineTask(Task[] list, String command) {
+        printBorder();
+        String[] commandToken = parseCommand(command, DELIMIT_COMMAND);
+
+        if (commandToken[1] != null) {
+            String[] taskDetails = parseCommand(commandToken[1], DELIMIT_DEADLINE);
+
+            if (taskDetails[1] != null) {
+                Task temp = new Deadlines(taskDetails[0], taskDetails[1]);
+
+                addTaskToList(list, temp);
+                printBorder();
+            }
+            else {
+                printError(MESSAGE_DEADLINE_USAGE);
+            }
+        }
+        else {
+            printError(MESSAGE_INVALID_ARGC);
+        }
+    }
+
+    /**
+     * Creates a new dated event task and appends it to the task list.
+     *
+     * @param list    a Task class list.
+     * @param command the details of the new dated task to be created.
+     */
+    public static void addEventTask(Task[] list, String command) {
+        printBorder();
+        String[] commandToken = parseCommand(command, DELIMIT_COMMAND);
+
+        if (commandToken[1] != null) {
+            String[] taskDetails = parseCommand(commandToken[1], DELIMIT_EVENT);
+
+            if (taskDetails[1] != null) {
+                Task temp = new Events(taskDetails[0], taskDetails[1]);
+
+                addTaskToList(list, temp);
+                printBorder();
+            }
+            else {
+                printError(MESSAGE_EVENT_USAGE);
+            }
+        }
+        else {
+            printError(MESSAGE_INVALID_ARGC);
+        }
+    }
+
+    public static void main(String[] args) {
+        String command;
+        Scanner in = new Scanner(System.in);
+        Task[] list = new Task[MAX_TASK];
+
+        greetings();
+        do {
```

Comment:
You may want to change this to a while loop because if the first command is bye then Bob would not understand.